src/main/java/com/jcabi/manifests/Manifests.java

```diff
@@ -237,6 +260,7 @@ public MfMap append(final Mfs streams) throws IOException {
                     this.attributes.put(attr.getKey(), attr.getValue());
                     ++saved;
                 }
+                this.addToMultiMap(attr.getKey(), attr.getValue());
```

Comment:
@pinaf Just to be clear, the tests will not pass with this line in any other position